### DISCUSSION TITLE
Image Prompting - Optimal Settings Question

### DISCUSSION THREAD

**Discussion by TrizOver9000** [lllyasviel/Fooocus#1859] [Q&A]
Greetings Fooocus peeps!

 I am curious if anyone has any suggestions on the optimal settings (styles, models, loras, weight ratios, etc.) for using image prompt to convert an animated image into realism? 

Appreciate any all responses!

**Comment by poor7** [comment]
Here, everything is individualized, but there are also general aspects to consider. If you give examples of images, it will be easier to suggest.

**Reply by TrizOver9000** [reply]
![GW walk](https://github.com/lllyasviel/Fooocus/assets/154026547/b961e052-2c36-4d98-92de-9065e8ec0562)

**Reply by TrizOver9000** [reply]
I've tried a handful of combinations but I can never seem to get a good result that honors the characters original image.

**Reply by xhoxye** [reply]
![QQ截图20240113090549](https://github.com/lllyasviel/Fooocus/assets/129571231/c8f51796-5ce9-4e27-a971-e0022784a846)
It's not the only number.
![QQ截图20240113090752](https://github.com/lllyasviel/Fooocus/assets/129571231/aa881ab6-b6c7-4ab5-ab68-9936aef9714e)
